x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -0,0 +1,659 @@
+                          id="xpack.security.accountManagement.userProfile.imageUrlLabel"
+                          defaultMessage="Image"
+                        />
+                      </FormLabel>
+                    }
+                    fullWidth
+                  >
+                    <FormField
+                      as={EuiFilePicker}
+                      name="data.avatar.imageUrl"
+                      value={undefined} /* EuiFilePicker breaks if value is provided  */
+                      initialPromptText={
+                        <FormattedMessage
+                          id="xpack.security.accountManagement.userProfile.imageUrlPromptText"
+                          defaultMessage="Select image file"
+                        />
+                      }
+                      onChange={createImageHandler((imageUrl) => {
+                        formik.setFieldValue('data.avatar.imageUrl', imageUrl ?? '');
+                      })}
+                      validate={{
+                        required: i18n.translate(
+                          'xpack.security.accountManagement.userProfile.imageUrlRequiredError',
+                          {
+                            defaultMessage: 'Upload an image.',
+                          }
+                        ),
+                      }}
+                      accept={IMAGE_FILE_TYPES.join(',')}
+                      display="default"
+                      fullWidth
+                    />
+                  </FormRow>
+                ) : (
+                  <EuiFlexGroup responsive={false} alignItems="center">
+                    <EuiFlexItem grow={false} style={{ width: 64 }}>
+                      <FormRow
+                        label={
+                          <FormLabel for="data.avatar.initials">
+                            <FormattedMessage
+                              id="xpack.security.accountManagement.userProfile.initialsLabel"
+                              defaultMessage="Initials"
+                            />
+                          </FormLabel>
+                        }
+                        fullWidth
+                      >
+                        <FormField
+                          name="data.avatar.initials"
+                          validate={{
+                            required: i18n.translate(
+                              'xpack.security.accountManagement.userProfile.initialsRequiredError',
+                              {
+                                defaultMessage: 'Enter initials.',
+                              }
+                            ),
+                            maxLength: {
+                              value: 2,
+                              message: i18n.translate(
+                                'xpack.security.accountManagement.userProfile.initialsMaxLengthError',
+                                {
+                                  defaultMessage: 'Enter no more than 2 characters.',
```

Comment:
Can the field limit the input to 2 characters so that this message doesn't have to pop up?